app/src/main/java/org/simple/clinic/home/report/ReportsEvent.kt

```diff
@@ -0,0 +1,3 @@
+package org.simple.clinic.home.report
```

Comment:
Can you create `ReportsUiActions` interface and extend in `ReportsUi` in no-op implementation ?

Reply:
There are no `UiActions ` in this screen, so we don't have to create it.